pages/activity/[id].js

```diff
@@ -1,52 +1,58 @@
 import TagList from "@/components/TagList/TagList";
 import styled from "styled-components";
-import Link from "next/link";
 import { data } from "@/lib/db/db";
 import { useRouter } from "next/router";
+import { StyledLink } from "@/components/StyledLink/StyledLink";
+import StyledReturn from "@/components/StyledReturn/StyledReturn";
+import { StyledContainer } from "@/components/StyledContainer/StyledContainer";
+import { StyledDeleteButton } from "@/components/StyledDeleteButton/StyledDeleteButton";
+import { Wrapper } from "@/components/Wrapper/Wrapper";
+import { StyledDate } from "@/components/StyledDate/StyledDate";
+import { TagImage } from "@/components/StyledTagImage/StyledTagImage";
 
-export default function SingleActivity({ editTag }) {
+export default function SingleActivity({ editTag, removeActivity }) {
```

Comment:
editTag is a function that has no functionality yet, you should remove it

```suggestion
export default function SingleActivity({ removeActivity }) {
```